bundle/manifests/observability-operator.clusterserviceversion.yaml

```diff
@@ -88,8 +92,23 @@ spec:
     - kind: ThanosRuler
       name: thanosrulers.monitoring.rhobs
       version: v1
-  description: Observability operator to manage Monitoring, Logging, Tracing for easier
-    observability.
+  description: |2+
+
+    Observability Operator is an operator created in Go to setup and manage highly available Monitoring Stack using Prometheus and Thanos Querier.
+    ### Supported Features
+    - Setup multiple Highly Available Prometheus monitoring stack
+    - Customizable configuration for managing Prometheus deployments
```

Comment:
Also supports Thanos Querier and Ruler deployment as well?

Reply:
I'd mention Alertmanager and Thanos Querier. It doesn't support Thanos Ruler directly.